README.md

```diff
@@ -1,20 +1,58 @@
 # 8080 Emulator and Space Invaders
 
-### Project Quickstart
-- Clone this repository
-- Install necessary packages:
-    - Required: gcc-10, g++-10, cmake (version 3.10+), SDL2, gtest
-    - Ubuntu/Debian/Mint: `sudo apt-get install cmake libgtest-dev libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev`
-    - Arch/Manjaro: `sudo pacman -Syyu cmake gtest sdl2 sdl2_image sdl2_mixer`
+## Table of Contents
+1. [Introduction](#Introduction) 
+    - [Dependencies](#Dependencies)
+    - [Required Build Tools](#Required Build Tools)
+    - [System Requirements](#System Requirements)
+    - [Installation](#Installation instructions for the library dependencies)
+2. [Project Quickstart](#Project Quickstart)
+    - [Play Space Invaders Game](#Play Space Invaders Game)
+    - [Running the Emulator](#Running the Emulator)
+    - [Speed Benchmarking and Speed Adjustment](#Speed Benchmarking and Speed Adjustment)
+    - [CPU Testing](#CPU Testing)
+3. [Reference materials and resources](#Reference materials and resources)
+## Introduction
+The Intel-8080 is an 8-bit microprocessor that Intel began producing in 1974. It could support clock rates up to 2 MHz. It has 8 main 8-bit registers, including an accumulator and a flags register. It also contains a 16-bit stack pointer and a 16-bit program counter. Furthermore, each pair of main registers (except the accumulator/flags registers) can be read/written as three 16-bit registers. It interfaces with the outside world through 40 pins. Included in these pins are 8 bidirectional data pins, 16 output address pins, two clock pins, an interrupt pin, a reset pin, and various other pins for power supply and indicating/controlling the processor state.
+
+Space Invaders is a 1978 arcade game by Taido. The game’s goal is to defeat five rows of eleven aliens that move horizontally back and force across the screen as they advance toward the bottom of the screen. 
```

Comment:
I know this is copy-pasted from something that I think I wrote, but we should correct Taido to Taito and "back and force" to "back and forth"

Reply:
oh good catch!!